pkg/reconciler/artifactbuild/artifactbuild.go

```diff
@@ -140,39 +141,39 @@ func (r *ReconcileArtifactBuild) handleTaskRunReceived(ctx context.Context, tr *
 		}
 	}
 	if len(ownerName) == 0 {
-		msg := "taskrun missing artifactbuilds ownerrefs %s:%s"
-		r.eventRecorder.Eventf(tr, corev1.EventTypeWarning, "MissingOwner", msg, tr.Namespace, tr.Name)
-		log.Info(msg, tr.Namespace, tr.Name)
+		msg := "pipelinerun missing artifactbuilds ownerrefs %s:%s"
+		r.eventRecorder.Eventf(pr, corev1.EventTypeWarning, "MissingOwner", msg, pr.Namespace, pr.Name)
+		log.Info(msg, pr.Namespace, pr.Name)
 		return reconcile.Result{}, nil
 	}
 
-	key := types.NamespacedName{Namespace: tr.Namespace, Name: ownerName}
+	key := types.NamespacedName{Namespace: pr.Namespace, Name: ownerName}
 	abr := v1alpha1.ArtifactBuild{}
 	err := r.client.Get(ctx, key, &abr)
 	if err != nil {
-		msg := "get for taskrun %s:%s owning abr %s:%s yielded error %s"
-		r.eventRecorder.Eventf(tr, corev1.EventTypeWarning, msg, tr.Namespace, tr.Name, tr.Namespace, ownerName, err.Error())
-		log.Error(err, fmt.Sprintf(msg, tr.Namespace, tr.Name, tr.Namespace, ownerName, err.Error()))
+		msg := "get for pipelinerun %s:%s owning abr %s:%s yielded error %s"
+		r.eventRecorder.Eventf(pr, corev1.EventTypeWarning, msg, pr.Namespace, pr.Name, pr.Namespace, ownerName, err.Error())
+		log.Error(err, fmt.Sprintf(msg, pr.Namespace, pr.Name, pr.Namespace, ownerName, err.Error()))
 		return reconcile.Result{}, err
 	}
 
 	//we grab the results here and put them on the ABR
-	for _, res := range tr.Status.TaskRunResults {
+	for _, res := range pr.Status.PipelineResults {
 		switch res.Name {
-		case TaskResultScmUrl:
+		case PipelineResultScmUrl:
 			abr.Status.SCMInfo.SCMURL = res.Value
-		case TaskResultScmTag:
+		case PipelineResultScmTag:
 			abr.Status.SCMInfo.Tag = res.Value
-		case TaskResultScmType:
+		case PipelineResultScmType:
 			abr.Status.SCMInfo.SCMType = res.Value
-		case TaskResultMessage:
+		case PipelineResultMessage:
 			abr.Status.Message = res.Value
-		case TaskResultContextPath:
+		case PipelineResultContextPath:
 			abr.Status.SCMInfo.Path = res.Value
 		}
 	}
 	if os.Getenv(DeleteTaskRunPodsEnv) == "1" {
```

Comment:
yeah our choice of env var name is now unfortunate

I'm assuming we would want to clean that up after the fact wrt this PRs change

Unfortunately it will be a staging exercise:
- add the new PR env in infra deployments after https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/jvm-build-service/operator-images.yaml#L16-L17 and https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/preview.sh#L100
- update the controller code here to add the new env var, but have the checks look at both env vars (if either is off, don't prune)
- update after https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml#L117 to add the new env var
- then start deleting refs to the tr env var, starting with infra-deployments, then jvm-build-service, then openshift/release